v2/internal/runner/runner.go

```diff
@@ -186,7 +186,7 @@ func New(options *types.Options) (*Runner, error) {
 	runner.hmapInputProvider = hmapInput
 
 	// Create the output file if asked
-	outputWriter, err := output.NewStandardWriter(!options.NoColor, options.NoMeta, options.Timestamp, options.JSON, options.JSONRequests, options.MatcherStatus, options.StoreResponse, options.Output, options.TraceLogFile, options.ErrorLogFile, options.StoreResponseDir)
+	outputWriter, err := output.NewStandardWriter(!options.NoColor, options.NoMeta, options.Timestamp, options.JSON, options.JSONRequests, options.MatcherStatus, options.StoreResponse, options.Output, options.TraceLogFile, options.ErrorLogFile, options.StoreResponseDir, options.Resume)
```

Comment:
As we are using mostly the `options` struct, we should consider about passing it directly as argument, the function signature would be much more concise, for example:
```go
// Create the output file if asked
outputWriter, err := output.NewStandardWriter(options)
```

Reply:
yes, much easier then to use, I do refactoring